assets/app/view/game/round/stock.rb

```diff
@@ -46,6 +46,7 @@ def render
 
           children.concat(render_corporations)
           children << h(Players, game: @game)
+          children << h(BuyCompanyFromOtherPlayer, game: @game) if @game.class::COMPANY_BUY_BETWEEN_PLAYERS
```

Comment:
you can get rid of all those return h(:div, '') if the actions returns buy company, it will be cleaner

Reply:
Yes, it became much cleaner after applying your suggestion